pkg/internal/approver/manager/review_test.go

```diff
@@ -217,7 +212,8 @@ func Test_Review(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			t.Cleanup(func() {
 				for _, obj := range test.policies {
-					if err := env.AdminClient.Delete(context.TODO(), &obj); /* #nosec G601 -- Func drops pointer at end of call. */ err != nil {
+					ctx := context.TODO() //nolint:usetesting
```

Comment:
I'm not sure how to fix this without suppression - the `t.Context` is already done when entering `t.Cleanup` functions.